pom.xml

```diff
@@ -60,50 +60,50 @@
 			<dependency>
 				<groupId>org.slf4j</groupId>
 				<artifactId>slf4j-api</artifactId>
-				<version>1.7.25</version>
+				<version>2.0.7</version>
 			</dependency>
 			<dependency>
 				<groupId>org.slf4j</groupId>
 				<artifactId>slf4j-simple</artifactId>
-				<version>1.7.25</version>
+				<version>2.0.7</version>
 			</dependency>
 			<dependency>
 				<groupId>org.apache.commons</groupId>
 				<artifactId>commons-lang3</artifactId>
-				<version>3.8.1</version>
+				<version>3.13.0</version>
 			</dependency>
 			<dependency>
 				<groupId>org.apache.commons</groupId>
 				<artifactId>commons-collections4</artifactId>
-				<version>4.2</version>
+				<version>4.4</version>
 			</dependency>
 			<dependency>
 				<groupId>org.apache.commons</groupId>
 				<artifactId>commons-text</artifactId>
-				<version>1.3</version>
+				<version>1.10.0</version>
 			</dependency>
 			<dependency>
 				<groupId>commons-io</groupId>
 				<artifactId>commons-io</artifactId>
-				<version>2.6</version>
+				<version>2.13.0</version>
 			</dependency>
 			<dependency>
 				<groupId>com.fasterxml.jackson.core</groupId>
 				<artifactId>jackson-databind</artifactId>
-				<version>2.11.3</version>
+				<version>2.15.2</version>
 			</dependency>
 
 			<!-- Testing -->
 			<dependency>
 				<groupId>junit</groupId>
 				<artifactId>junit</artifactId>
-				<version>4.13.1</version>
+				<version>4.13.2</version>
 				<scope>test</scope>
 			</dependency>
 			<dependency>
 				<groupId>org.assertj</groupId>
 				<artifactId>assertj-core</artifactId>
-				<version>2.9.1</version><!-- cannot user newer than 2.x with java version 7 -->
+				<version>3.24.2</version><!-- cannot user newer than 2.x with java version 7 -->
```

Comment:
I would like to keep the Socomo code, including its dependencies, at version 7 of Java. The purpose is that Socomo is often used as a tool to understand old projects that you've somehow inherited and have to learn and maintain. Those old projects still sometime use Java 7 (and often Java 8). So I would like that those projects can still use JDK 7 to execute Socomo. That's why I keep libraries versions very low.

I've seen that on new JDKs the old assertj-core complains about lack of access via reflection. But this can be solved by adding `--add-opens java.base/java.lang=ALL-UNNAMED` to surefire configuration (as you did). Any other reason that requires upgrading those libraries?